operations/deployment-engineering/frequently-asked-questions.md

```diff
@@ -0,0 +1,66 @@
+
+## Who owns long-term maintenance of solutions built by this team?
+
+Most solutions should not require long-term maintenance, as they will be standalone scripts, plugins or other code that is not dependent on Mattermost server versions.
+
+For solutions that require maintenance, we will discuss ownership between Deployment Engineering, TAMs, R&D and a vendor on a case-by-case basis.
+
+## How does this role differ from TAMs (Technical Account Managers)?
+
+TAMs are responsible for ongoing relationship management with technical champions, and own pre-defined customer onboarding in standard deployment environments (ie. follow our gold standard deployments).
+
+On the other hand, Deployment Engineering supports non-standard deployments with complex technical problems, as well as work closely with customers on temporary one-off custom solutions & integrations based on [requirements outlined by the CSMs/TAMs](https://docs.google.com/document/d/1EbP4Ab7N2hEAv--DRCcZHk0XtlG1MFP8F8F0viaTM7I/edit).
+
+Thus, Deployment Engineering acts as an extension of R&D supporting post-sales and customer success on the most complex, technical problems, with expertise on how to run and deploy Mattermost and the platform as a whole.
+
+Finally, we encourage a representative from TAM to join weekly Deployment Engineering meetings, and vice versa, to help deepen collaboration & understanding of each team’s workstreams.
+
+## Why does this role not report to Customer Success, like TAMs?
+
+TAMs are responsible for ongoing relationship management, and are evaluated against renewal rates.
+
+Deployment Engineering engages with customers on a short-term basis, in a technical capacity. Their evaluation is not based on renewal rates, and hence do not directly report to the revenue organization (Sales & Customer Success).
+
+## How does this role differ from Support Engineers & Customer Reliability Engineers?
+
+Support & Customer Reliability Engineers provide first-line of support & troubleshooting, and may fix bugs & security issues, while Deployment Engineering is a point of contact for critical deployment escalations only for customers over $50k ARR (e.g. performance degradation).
+
+## Why does this role not report to Customer Support, if they work on escalations?
+
+Support Engineers are evaluated against CSAT scores and SLA coverage. Deployment Engineering is evaluated against Customer Seats Deployed.
+
+Moreover, Deployment Engineering is not a support-function, but instead their primary ownership is to help customers run and deploy Mattermost at scale.
+
+## Will this team be a call or SET rotation?
+
+No.
+
+## Why is this team important now?
```

Comment:
I think this section should go along with how it is measured so it stresses the importance and meaning of the metric.

Reply:
Thanks! Moved it together with the 'how is this team measured' section